build.gradle

```diff
@@ -178,25 +187,39 @@ task downloadMcJars(dependsOn: downloadWantedVersionManifest) {
 	}
 }
 
-task downloadIntermediary(type: Download){
+task extractIntermediary(type: FileOutput) {
 	group = buildMappingGroup
-	def url = "https://github.com/FabricMC/intermediary/raw/master/mappings/${minecraft_version}.tiny"
-	src UrlEscapers.urlFragmentEscaper().escape(url)
-	dest new File(cacheFilesMinecraft, "${minecraft_version}-intermediary.tiny")
+
+	def v1Input = configurations.intermediary.files[0]
```

Comment:
This seems a bit fragile? Would it be a good idea to have a v1 and v2 confuration?

Reply:
The v2 intermediary is actually not needed at all because I moved it to loom

Reply:
look at the new commit

Reply:
If you do get only 1 file, use guava Iterable getOnlyElement which throws when it detects multiple possible entries.